app/src/main/java/org/schabi/newpipe/download/DownloadDialog.java

```diff
@@ -684,6 +684,9 @@ private void prepareSelectedDownload() {
         prefs.edit()
                 .putString(getString(R.string.last_used_download_type), selectedMediaType)
                 .apply();
+
+        Toast.makeText(context, getString(R.string.download_has_started),
+                Toast.LENGTH_LONG).show();
```

Comment:
This is something someone with a better eye for UX can answer, but i don't think `LENGTH_LONG` (3.5 seconds) is necessary. `LENGTH_SHORT` (2 seconds) should be enough in my view.

https://stackoverflow.com/questions/7965135/what-is-the-duration-of-a-toast-length-long-and-length-short

Reply:
You are right. I tried both and i think that `LENGTH_SHORT` looks more consistent. I'm changing it now.